exhaust.py

```diff
@@ -18,6 +18,7 @@
 MANDATORY_CONSTRAINTS = {
     "vivado": "xdc",
     "vpr": "pcf",
+    "nextpnr": "xdc",
```

Comment:
Can nextpnr read xdc now? I thought you have to specify constraints using a Python language thingy?

Reply:
As far as I have seen in the nextpnr-xilinx examples, xdc files are used to constraint IO pads